### PR TITLE
examples: Remove http2 options/protocol from config where not required

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: examples: Remove http2 options/protocol from config where not required
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
follow on from #15563 

initially i have just removed all and will readd the http2 options to any that fail ci

**Comment by phlax** [comment]
the `literalincludes` will need to be checked as their line nums may have changed slightly

**Review by davinci26 — DISMISSED** [review]
LGTM, the configs are cleaner that way.

File: docs/root/start/sandboxes/dynamic-configuration-filesystem.rst [hunk 1]
```diff
@@ -90,8 +90,8 @@ from ``service1`` to ``service2``:
 .. literalinclude:: _include/dynamic-config-fs/configs/cds.yaml
    :language: yaml
    :linenos:
-   :lines: 7-15
-   :lineno-start: 7
+   :lines: 6-14
+   :lineno-start: 6
```
Comment (phlax) [inline docs/root/start/sandboxes/dynamic-configuration-filesystem.rst:94]: @mattklein123 just to highlight this - if configs in the examples dir are changed we need to check that they arent included in docs with line highlights etc

there arent many of them - they can be found by grepping  for `literalinclude` and `.yaml`

**Review by mattklein123 — COMMENTED** [review]
Thanks. Do we still have some example that shows HTTP/2? Should we specifically call that out in the docs? I assume the gRPC ones do, etc.?

/wait-any

**Comment by phlax** [comment]
> Do we still have some example that shows HTTP/2?

yep

> Should we specifically call that out in the docs?

i wondered about this - either a sandbox - or something in getting started

ive added a ticket regarding this #15605 

> I assume the gRPC ones do, etc.?

yep

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Pull request merged**